document-store/src/main/java/org/hypertrace/core/documentstore/postgres/query/v1/parser/filter/PostgresRelationalFilterParserFactory.java

```diff
@@ -0,0 +1,9 @@
+package org.hypertrace.core.documentstore.postgres.query.v1.parser.filter;
+
+import org.hypertrace.core.documentstore.expression.impl.RelationalExpression;
+import org.hypertrace.core.documentstore.postgres.query.v1.parser.filter.PostgresRelationalFilterParser.PostgresRelationalFilterContext;
+
+public interface PostgresRelationalFilterParserFactory {
+  PostgresRelationalFilterParser parser(
+      final RelationalExpression expression, final PostgresRelationalFilterContext context);
```

Comment:
if `context` is not required, we can remove it from an argument.

Reply:
Sure, yeah.